aws/components/apigateway/setup.ftl

```diff
@@ -303,21 +303,57 @@
             outputs={}
             dependencies=apiId
         /]
+
+        [#-- Throttling Configuration --]
+        [#local openapiIntegrations = 
+            getOccurrenceSettingValue(occurrence, [["apigw"], ["Integrations"]], true)]
+
+        [#local methodSettings = [
+            {
+                "HttpMethod": "*",
+                "ResourcePath": "/*",
+                "LoggingLevel": "INFO",
+                "DataTraceEnabled": true
+            } +
+            attributeIfContent(
+                "ThrottlingBurstLimit", 
+                openapiIntegrations.Throttling.BurstLimit) +
+            attributeIfContent(
+                "ThrottlingRateLimit",
+                openapiIntegrations.Throttling.RateLimit)
+        ]]
+
+        [#-- Integration Patterns (as Regex) into Matching Method Throttling (as explicit paths) --]
+        [#if definitionsObject[core.Id]?? ]
+            [#list definitionsObject[core.Id].paths as path,pathConfig]
+                [#list pathConfig?keys as verb]
+                    [#list openapiIntegrations.Patterns as pattern]
+                        [#if path?matches(pattern.Path)] 
+                            [#local methodSettings = combineEntities(
```

Comment:
can just do array addition - I think it would be clearer.

Reply:
Updated